client/scss/components/_header.scss

```diff
@@ -129,7 +129,7 @@
       }
     }
 
-    .left {
+    .w-header__primary {
       float: left;
```

Comment:
as per description on the issue - the plan is to replace the floats/clear with a more suitable approach such as flexbox.